x/ccv/provider/keeper/relay.go

```diff
@@ -300,24 +309,65 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 // EndBlockCIS contains the EndBlock logic needed for
 // the Consumer Chain Removal sub-protocol
 func (k Keeper) EndBlockCCR(ctx sdk.Context) {
-	currentTime := uint64(ctx.BlockTime().UnixNano())
+	currentTime := ctx.BlockTime()
+	currentTimeUint64 := uint64(currentTime.UnixNano())
 
 	// iterate over initTimeoutTimestamps
-	var removedChainIds []string
+	var chainIdsToRemove []string
 	k.IterateInitTimeoutTimestamp(ctx, func(chainID string, ts uint64) bool {
```

Comment:
Why is the time coming in as a uint64?

Reply:
The init timeout timestamp is saved as an uint64